go.mod

```diff
@@ -4,9 +4,9 @@ go 1.17
 
 require (
 	github.com/fsnotify/fsnotify v1.5.1
-	github.com/go-logr/logr v0.4.0
-	go.uber.org/atomic v1.8.0 // indirect
-	go.uber.org/multierr v1.7.0 // indirect
+	github.com/go-logr/logr v1.2.2
+	go.uber.org/atomic v1.9.0 // indirect
+	go.uber.org/multierr v1.8.0 // indirect
 	go.uber.org/zap v1.21.0
 	k8s.io/api v0.21.9
```

Comment:
Update is available https://github.com/kubernetes/api/tree/v0.21.10

Reply:
done